CMakeLists.txt

```diff
@@ -3,6 +3,12 @@ set(CMAKE_CXX_STANDARD 14)
 
 project(client)
 
+find_package(ECM 5.50.0 NO_MODULE)
```

Comment:
This effectively adds ECM as a new dependency. I'm not necessarily against it (on the contrary, I find it a shame we don't to be honest) but that needs to be properly coordinated. Indeed, it's not available by default on Windows or Mac for instance so our build nodes would need it first and client-building would need to clone it, etc.

This is I think why one would find copies of parts of ECM in our own cmake directory. And I guess in the case of the inotify related part it might be "just" about updating our copy with the latest from ECM (for now).

From what I see in the rest of your changes, it looks like we're not really using much more than before from it. Can we make it without that line or am I mistaken?

Reply:
Yes sure, we can just use the FindINotify from ecm. Then this line can be removed.